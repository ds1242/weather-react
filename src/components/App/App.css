*{
    margin:0;
    padding:0;
    --primary_color:#80A1C1;
    --secondary_color: #F6F1D1;
    --tertiary_color: #285238;
    --quartiary_color: #384E77;
    --cinquieme_color: #EAD637;
    /* box-sizing: border-box; */
}


