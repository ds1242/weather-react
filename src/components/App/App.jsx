import React from 'react';
import Nav from '../Nav';
import Footer from '../Footer';
import './App.css';

function App() {
  return (
    <div className="App">
      <Nav></Nav>
      <div>hello world</div>
      <Footer></Footer>
    </div>
  );
}

export default App;
