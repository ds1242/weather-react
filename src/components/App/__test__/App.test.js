import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import App from '../App.jsx';

afterEach(cleanup);

describe('App component',() => {
  it('renders', () => {
    render(<App />);
  });
  it('matches snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});