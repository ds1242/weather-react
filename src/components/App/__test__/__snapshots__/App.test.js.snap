// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <nav>
      Hello Navbar
    </nav>
    <h1>
      hello world
    </h1>
  </div>
</DocumentFragment>
`;
